src/ec/packages/ec-component-hero-banner/hero-banner.story.js

```diff
@@ -15,105 +16,97 @@ import heroBanner from './ecl-hero-banner.html.twig';
 import notes from './README.md';
 
 const PrepareBanner = data => {
+  getExtraKnobs(data);
+  if (data.link.link.label) {
+    data.link.link.label = text(
+      'Link label',
+      data.link.link.label,
+      buttonLabels.required
+    );
+  }
+
+  if (data.title) {
+    data.title = text('Title', data.title, buttonLabels.required);
+  }
+
+  if (data.title) {
+    data.title = text('Title', data.title, buttonLabels.required);
```

Comment:
You see also that the effort is to use labels for the knobs that are corresponding to the right piece of information in the input data, this property is title, not Title and so its label should be ;)